src/Native/build.proj

```diff
@@ -72,6 +72,9 @@
       <PackageRid>$(PackageRid)-$(TargetArchitecture)</PackageRid>
     </PropertyGroup>
 
+    <Copy SourceFiles="$(PackagesDir)mlnetmkldeps\$(MlNetMklDepsPackageVersion)\runtimes\$(PackageRid)\native\$(NativeLibPrefix)MklImports$(NativeLibExtension)"
```

Comment:
I think it would be good to add a comment here to describe what we are doing.

"Copy MklImports into the native assets folder, so it can be treated like a regularly built native assembly."